doc/changelog.md

```diff
@@ -311,22 +463,22 @@
   register.
   [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
 
-* Correctly querying batching rules for `jax.scipy.linalg.expm`
+* `jax.scipy.linalg.expm` is now supported within qjit-compiled functions.
   [(#733)](https://github.com/PennyLaneAI/catalyst/pull/733)
+  [(#752)](https://github.com/PennyLaneAI/catalyst/pull/752)
 
-* Correctly linking openblas routines necessary for `jax.scipy.linalg.expm`.
+  This required correctly linking openblas routines necessary for `jax.scipy.linalg.expm`.
   In this bug fix, four openblas routines were newly linked and are now discoverable by
   `stablehlo.custom_call@<blas_routine>`. They are `blas_dtrsm`, `blas_ztrsm`, `lapack_dgetrf`,
   `lapack_zgetrf`.
-  [(#752)](https://github.com/PennyLaneAI/catalyst/pull/752)
 
-* Correctly recording types of constant array when lowering `catalyst.grad` to mlir
+* Correctly recording types of constant array when lowering `catalyst.grad` to mlir.
   [(#778)](https://github.com/PennyLaneAI/catalyst/pull/778)
 
 * Callbacks can now return types which can be flattened and unflattened.
   [(#812)](https://github.com/PennyLaneAI/catalyst/pull/812)
 
-* `catalyst.qjit` and `catalyst.grad` can now get `__name__` from `functools.partial`.
+* `catalyst.qjit` and `catalyst.grad` now get `__name__` from `functools.partial`.
   [(#820)](https://github.com/PennyLaneAI/catalyst/pull/820)
```

Comment:
@tzunghanjuang could you describe what this change was for?

Reply:
It's to make catalyst able to get function's name. For cases that calculate `grad` from functions wrapped by partial trigger no attribute `__name__` error.
```python
        def fn(x, y):
            return x * y

        partial_fn = partial(fn, y=1)

        @qjit
        def grad_partial_fn(x):
            return grad(partial_fn)(x)
```

Reply:
```suggestion
* `catalyst.qjit` and `catalyst.grad` now work correctly with
  functions that have been wrapped with `functools.partial`.
  [(#820)](https://github.com/PennyLaneAI/catalyst/pull/820)
```